core/dbt/parser/schema_test_builders.py

```diff
@@ -303,15 +311,25 @@ def get_test_name(self) -> Tuple[str, str]:
             name = '{}_{}'.format(self.namespace, name)
         return get_nice_schema_test_name(name, self.target.name, self.args)
 
+    def construct_config(self) -> str:
+        configs = ",".join([
+            f"{key}=" + (f"'{value}'" if isinstance(value, str) else str(value))
```

Comment:
@kwigley This is actually the line that fixed the `Undefined` Jinja error, because I was accidentally passing the config down on line 329 as:
```
{{ config(severity = warn) }}
```
Instead of `severity = 'warn'`. At the same time, we can't quote values that need to be non-string types (namely, `enabled` is boolean). This is a pretty janky way of writing that, I'm sure there's a more pythonic / f-string-smart way to do it.